### PR TITLE
Improve the JS performance for logic-heavy stylesheets

### PR DESCRIPTION
**Pull request by nex3**
See #113

### CODE REVIEW

File: lib/src/visitor/async_evaluate.dart [hunk 1]
```diff
@@ -1962,15 +1962,19 @@ class _EvaluateVisitor
...
     var named = await normalizedMapMapAsync<String, Expression, Value>(
         arguments.named,
         value: (_, expression) => expression.accept(this));
 
-    var positionalNodes =
-        trackSpans ? arguments.positional.map(_expressionNode).toList() : null;
+    var positionalNodes = trackSpans
+        ? [
+            for (var expression in arguments.positional)
+              _expressionNode(expression)
```
Comment (srawlins) [inline lib/src/visitor/async_evaluate.dart]: Good that this new syntax is performant!

File: tool/grind/npm.dart [hunk 2]
```diff
@@ -40,7 +40,7 @@ void _js({@required bool release}) {
     // * We expect our test coverage to ensure that nothing throws subtypes of
     //   Error.
     // * We thoroughly test edge cases in user input.
-    if (release) ...["-O4", "--fast-startup"]
+    if (release) ...["-O4", "--no-minify", "--fast-startup"]
```
Comment (srawlins) [inline tool/grind/npm.dart]: Should we comment on why we pass `no-minify`?
